sql/core/src/test/resources/sql-tests/inputs/describe_tbleproperty_validation.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE table_with_comment (a STRING, b INT) COMMENT 'actual comment';
+
+DESC formatted table_with_comment;
+
+-- ALTER TABLE BY MODIFYING COMMENT
+ALTER TABLE table_with_comment set tblproperties(comment = "modified comment");
+
+DESC formatted table_with_comment;
+
+-- DROP TEST TABLE
+DROP TABLE table_with_comment;
+
+-- CREATE TABLE WITHOUT COMMENT
+CREATE TABLE table_comment (a STRING, b INT);
+
+DESC formatted table_comment;
+
+-- ALTER TABLE BY ADDING COMMENT
+ALTER TABLE table_comment set tblproperties(comment = "added comment");
+
+DESC formatted table_comment;
+
+-- DROP TEST TABLE
+DROP TABLE table_comment;
```

Comment:
The basic function has been tested in the other suites. How about changing this test suite to `describe-table-after-alter-table`?
```SQL
CREATE TABLE table_comment (a STRING, b INT)

-- ALTER TABLE BY ADDING COMMENT
ALTER TABLE table_comment set tblproperties(comment = "added comment");

DESC formatted table_comment;

-- ALTER TABLE BY MODIFYING COMMENT
ALTER TABLE table_comment set tblproperties(comment = "modified comment");

DESC formatted table_comment;

-- DROP TEST TABLE
DROP TABLE table_comment;
```